robocop/checkers/naming.py

```diff
@@ -743,3 +756,43 @@ def check_if_keyword_is_deprecated(self, keyword_name, node):
                 col=col,
                 version=f"{ROBOT_VERSION.major}.*",
             )
+
+
+class SpacesBeforeArgumentChecker(VisitorChecker):
+    reports = ("missing-4-spaces-before-argument",)
+
+    def visit_KeywordCall(self, node):
+        if node.keyword in KNOWN_KEYWORDS:
+            return
+        try:
+            known_keyword = next(keyword for keyword in KNOWN_KEYWORDS if node.keyword.startswith(f"{keyword} "))
```

Comment:
We need to normalize name to also catch:
```
click_element
ClickElement
```
Most likely KNOWN_KEYWORDS should contain already normalized names, then on visit_KeywordCall we normalize the keyword name and compare.